test/e2e/e2e_test.go

```diff
@@ -62,6 +62,35 @@ var _ = Describe("For a Kubernetes cluster with Tekton and build installed", fun
 
 			validateBuildRunToSucceed(namespace, br)
 		})
+
+	})
+
+	Context("when a Buildpacks build is defined with contextDir", func() {
+
+		It("successfully runs a build with nodejs runtime", func() {
+			createBuild(namespace, "buildpacks-v3-nodejs", "test/data/build_buildpacks-v3_nodejs_cr.yaml")
```

Comment:
I think this new test is just for contextDir, we don't need to cover more runtimes.

And we already have the node js test for buildpacks test:
https://github.com/redhat-developer/build/blob/master/test/e2e/e2e_test.go#L52-L65

Do we still need another nodejs test for contextDir? It will cost more time.

If the new added tests are used to make sure all runtimes are verified for buildpacks, can we add an e2e flag to choose test them or not.

Reply:
why you think we dont need to cover more runtimes. How can then you be sure the buildpacks work for other runtimes?

> Do we still need another nodejs test for contextDir? It will cost more time.

I can remove that. 

> If the new added tests are used to make sure all runtimes are verified for buildpacks, can we add an e2e flag to choose test them or not.

We could discuss this on another issue, but ideally you want always to test all runtimes.

Reply:
And one will not need new e2e flags because one can solve this with the Ginkgo `focus` and `skip` flags already.

Reply:
Is there way to just skip this new `Context("when a Buildpacks build is defined with contextDir", func()` e2e tests by using the Ginkgo parameter?

I remember I also heard from @otaviof ,

The user may have different requirements for different buildstratgies.

Such as I just want to enable the Dockerfile build in my env, while you just want to use buildpacks in your env.

So it is better, we can have different e2e test suites, so that we can use Ginkgo parameter to just run the tests what I want, then I am fine for that.

Reply:
I am fine now if we could have an issue later to track run different test suites by maybe Ginkgo parameter

Reply:
To just skip based on `contextDir`:

```bash
export TEST_E2E_FLAGS="-failFast -p -randomizeAllSpecs -slowSpecThreshold=300 -timeout=15m -trace -v -skip=contextDir"
make test-e2e
```

But typically, one rather wants to run a selective set of tests, like:

```bash
export TEST_E2E_FLAGS="-failFast -p -randomizeAllSpecs -slowSpecThreshold=300 -timeout=15m -trace -v -focus=Kaniko"
make test-e2e
```

Reply:
yes, @SaschaSchwarze0 is correct, the `skip` is the key for your requirement @zhangtbj , if you agree, pls resolve this converstation.

Reply:
Very cool, thanks!